.changesets/maint_garypen_fix_telemetry_integration_test_macos.md

```diff
@@ -0,0 +1,5 @@
+### fix annoying integration test warning on macOS ([PR #4919](https://github.com/apollographql/router/pull/4919))
+
+For a little while we have had a warning with integration tests on macOS. This fixes the warning.
```

Comment:
Can we provide the warning message that was fixed? Then we could update the changelog like this:

```suggestion
### Fix integration test warning on macOS ([PR #4919](https://github.com/apollographql/router/pull/4919))

Previously, integration tests of the router on macOS could produce the warning message `TBD`. That issue has been resolved. 
```